python/ambassador/envoy/v3/v3ready.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2021 Datawire. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+import os
+
+from typing import List, TYPE_CHECKING
+
+from .v3listener import V3Listener
+
+if TYPE_CHECKING:
+    from . import V3Config # pragma: no cover
+
+# The defaults can be changed by using those 2 env vars:
+# AMBASSADOR_READY_PORT: Port number (default 8002)
+# AMBASSADOR_READY_LOG: true/false (default true)
+ambassador_ready_port = int(os.getenv("AMBASSADOR_READY_PORT", "8002"))
+if ambassador_ready_port not in range(1, 32767):
+    ambassador_ready_port = 8002
+ambassador_ready_log = os.getenv("AMBASSADOR_READY_LOG", "true") in ("yes", "true", "t", "1")
```

Comment:
In `ambassador/utils.py` there's a `parse_bool` that does exactly this, so

```
ambassador_ready_log = parse_bool(os.getenv("AMBASSADOR_READY_LOG", "true"))
```

is probably a better choice.